README.md

```diff
@@ -1,17 +1,15 @@
 Habitat Depot
 =============
 
-> Use at your own risk!
```

Comment:
Keep this or rewrite as a mentions that Private Depots is not a formal product yet and there's no declared support from Chef.